### PR TITLE
Remove screenshots from Grafana tutorial

### PR DESCRIPTION
**Pull request by NHingerl**
**Description**

Changes proposed in this pull request:

- remove obsolete Console screenshot
- review content - click-level instructions are high-maintenance, especially when describing another project's GUI.
- remove Grafana screenshots - again, high maintenance effort to check regularly whether the GUI is updated.

**Related issue(s)**

#11228

### CODE REVIEW

File: docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md [hunk 1]
```diff
@@ -12,76 +12,46 @@ Follow these sections to create the Gauge dashboard type for the `cpu_temperatur
...
 
-   ![Red dashboard](./assets/red-dashboard.png)
+- It turns **red** if the current metric value ranges from 75 to 90 degrees Celsius:
 
-You can also define the dashboard's ConfigMap and add it to the `resources` folder under the given component's chart. To make the dashboard visible, simply use the `kubectl apply` command to deploy it. For details on adding monitoring to components, see the [`README.md`](https://github.com/kyma-project/kyma/blob/master/resources/monitoring/charts/grafana/README.md) document.
+## Next steps
 
-> **NOTE:** If you don't want to proceed with the following tutorial, [clean up the configuration](obsv-05-clean-up-configuration.md).
+- Follow the tutorial to [Define alerting rules](obsv-04-define-alerting-rules-monitor.md).
+- You can also define the dashboard's ConfigMap and add it to the `resources` folder under the given component's chart. To make the dashboard visible, simply use the `kubectl apply` command to deploy it. For details on adding monitoring to components, see the [`README.md`](https://github.com/kyma-project/kyma/blob/master/resources/monitoring/charts/grafana/README.md) document.
```
Comment (majakurcius) [inline docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md:56]:
```suggestion
- You can also define the dashboard's ConfigMap and add it to the `resources` folder under the given component's chart. To make the dashboard visible, simply use the `kubectl apply` command to deploy it. For details on adding monitoring to components, see the [Grafana chart `README.md`](https://github.com/kyma-project/kyma/blob/master/resources/monitoring/charts/grafana/README.md).
```
I feel like just `README` is not descriptive enough. 
I'd phrase it `see the Grafana chart README` or as in the suggestion above.

File: docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md [hunk 2]
```diff
@@ -12,76 +12,46 @@ Follow these sections to create the Gauge dashboard type for the `cpu_temperatur
...
+6. Under **Panel > Display**, make sure the threshold labels and threshold markers are activated to display this range on the dashboard.
 
-   ![Panel title](./assets/enable-labels.png)
-
-7. Click the save button in the top right corner of the page to save the changes. Add an optional note to describe the changes made.
-
-   ![Note](./assets/save-note.png)
+7. Save your changes. We recommend that you add a note to describe the changes made.
 
 ## Verify the dashboard
```
Comment (majakurcius) [inline docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md:45]:
```suggestion
## Verify the changes
```
or `Verify the new settings` or `Check the dashboard`? You're not really verifying that the dashboard exists, but rather that the changes have actually been saved and the new settings are being applied.

File: docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md (hunk 1 above)
Comment (NHingerl) [reply docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md:56]: fully agree.

File: docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md (hunk 2 above)
Comment (NHingerl) [reply docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md]: You're absolutely right. I'll try "Check the dashboard".

File: docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md [hunk 3]
```diff
@@ -2,86 +2,56 @@
...
 
-You have performed the steps to observe application metrics using the `monitoring-custom-metrics` example and successfully deployed the `sample-metrics-8081` service which exposes the `cpu_temperature_celsius` metric.
+You have performed the steps to observe application metrics using the `monitoring-custom-metrics` example and successfully deployed the `sample-metrics-8081` service that exposes the `cpu_temperature_celsius` metric.
 
 Follow these sections to create the Gauge dashboard type for the `cpu_temperature_celsius` metric.
 
 ## Create the dashboard
 
-1. Navigate to Grafana. It is available under the `https://grafana.{DOMAIN}` address, where `{DOMAIN}` is the domain of your Kyma cluster, such as `https://grafana.kyma.example.com` or `https://grafana.local.kyma.dev/`. To access it from the Console UI, click **Metrics** on the left navigation menu.
+1. [Access Grafana](../../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md).
```
Comment (majakurcius) [inline docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md:15]: The link's not working

**Review by majakurcius — APPROVED** [review]

**Pull request merged**